apps/fortuna/config.sample.yaml

```diff
@@ -32,6 +32,9 @@ chains:
     # How much to charge in fees
     fee: 1500000000000000
 
+    # Set this temporarily to false if you have changed the fees and want to apply a new baseline fee.
+    sync_fee_only_on_register: true
```

Comment:
I think this is a brittle config flag -- people will forget to unset it. I think it is better to auto-detect whether or not you need to set the fees differently.

E.g., can we run the same code we run in the keeper to set the fees based on the gas price?